ogre/src/CMakeLists.txt

```diff
@@ -35,12 +37,24 @@ target_link_libraries(${ogre_target}
 # Build the unit tests
 ign_build_tests(TYPE UNIT SOURCES ${gtest_sources} LIB_DEPS ${ogre_target})
 
+# Note that plugins are currently being installed in 2 places: /lib and the engine-plugins dir
+install(TARGETS ${ogre_target} DESTINATION ${IGNITION_RENDERING_ENGINE_INSTALL_DIR})
+
 if(WIN32)
   # tests needs .dll in the same directory
   add_custom_command(TARGET ${ogre_target} POST_BUILD
     COMMAND ${CMAKE_COMMAND} -E copy
     $<TARGET_FILE:${ogre_target}> ${CMAKE_CURRENT_BINARY_DIR})
 endif()
 
+set (versioned ${CMAKE_SHARED_LIBRARY_PREFIX}${PROJECT_NAME_LOWER}-${engine_name}${CMAKE_SHARED_LIBRARY_SUFFIX})
+set (unversioned ${CMAKE_SHARED_LIBRARY_PREFIX}${PROJECT_NAME_NO_VERSION_LOWER}-${engine_name}${CMAKE_SHARED_LIBRARY_SUFFIX})
+
+if (WIN32)
+  # create_symlink requires cmake 3.13 on windows
+  cmake_minimum_required(VERSION 3.13 FATAL_ERROR)
+endif()
+EXECUTE_PROCESS(COMMAND ${CMAKE_COMMAND} -E create_symlink ${versioned} ${unversioned})
+INSTALL(FILES ${PROJECT_BINARY_DIR}/${unversioned} DESTINATION ${IGNITION_RENDERING_ENGINE_INSTALL_DIR})
```

Comment:
[windows build](https://build.osrfoundation.org/job/ign_rendering-pr-win/1015/) failed as it has problem with installing sym links. I think the dartsim plugin is not built on windows so jenkins does not catch the problem but we ran into this with TPE and had to do a copy instead of a symlink on windows:
https://github.com/ignitionrobotics/ign-physics/blob/master/tpe/plugin/CMakeLists.txt#L42-L51

Reply:
As a reference, we decided to use the `cmake_minimum_required(VERSION 3.13 FATAL_ERROR)` approach on [this PR](https://osrf-migration.github.io/ignition-gh-pages/#!/ignitionrobotics/ign-gazebo/pull-requests/222/page/1#comment-94939048). I guess it doesn't work afterall. CC @scpeters

Reply:
I thought it requires special privileges on windows to create symlinks:

* https://osrf-migration.github.io/ignition-gh-pages/#!/ignitionrobotics/ign-gazebo/pull-requests/222/page/1#comment-94939827
* https://www.howtogeek.com/howto/16226/complete-guide-to-symbolic-links-symlinks-on-windows-or-linux/

Reply:
I added the build rule for windows that was in TPE, I guess as we work on Windows compatibility going forward in Q4, I can do more native Windows build testing in the next few months?

Reply:
sounds good. We need a bit of help on the Windows side..!